repairnator/repairnator-pipeline/src/main/java/fr/inria/spirals/repairnator/process/step/AbstractStep.java

```diff
@@ -267,7 +267,8 @@ protected void cleanMavenArtifacts() {
             }
         }
 
-        if (this.inspector.getRepoLocalPath() != null && this.config.isClean()) {
+        File repoDir = new File(this.inspector.getRepoLocalPath() + File.separator + Utils.REMOTE_REPO_EXT);
```

Comment:
Prefer using `new File(children, repo);` here it's `new File(this.inspector.getRepoLocalPath(), Utils.REMOTE_REPO_EXT)`

Reply:
Done.